### PR TITLE
Expose requesterLocId from request.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#670

### CODE REVIEW

File: packages/client/src/Loc.ts [hunk 1]
```diff
@@ -429,6 +429,7 @@ export abstract class LocRequestState extends State {
         return {
             ...request,
             requesterAddress: request.requesterAddress || undefined,
+            requesterLocId: request.requesterIdentityLoc ? new UUID(request.requesterIdentityLoc) : undefined,
```
Comment (gdethier) [inline packages/client/src/Loc.ts:432]: I am not sure we are enforcing backend-side that only LOs can actually create logion LOCs.

**Review by gdethier — APPROVED** [review]
:+1:

**Review by gdethier — APPROVED** [review]
:+1:

File: packages/client/src/Loc.ts (hunk 1 above)
Comment (benoitdevos) [reply packages/client/src/Loc.ts:432]: As discussed, this check is done by the node runtime. On the backend, we still check that the request is submitted by either the LO or the requester. For transactions/collections we also check there is a valid identity LOC. Those checks are sufficient IMHO.

**Pull request merged**